Stratis.Bitcoin.Tests/BlockStore/LoopTests/BlockStoreLoopStepBaseTest.cs

```diff
@@ -15,6 +15,9 @@
 
 namespace Stratis.Bitcoin.Tests.BlockStore.LoopTests
 {
+    /// <summary>
+    /// Base test class for all the BlockStoreLoop tests
+    /// </summary>
     public class BlockStoreLoopStepBaseTest
```

Comment:
The formality is unit tests don't need the documentation rules.
Ofcourse if you want to comment you are welcome to.